README.md

```diff
@@ -167,6 +167,50 @@ It is possible to generate a minimal SHACL shape given a `Cube` and a set of `Ob
 
 SPARQL CONSTRUCT queries will be provided in this repository.
 
+### Types of Dimensions
+
+To be able to understand the nature of a dimension we can type the dimension in the constraints. In general we have two mandatory types, the `cube:MeasureDimension` and the `cube:KeyDimension`. Whereas the `cube:MeasureDimension` does flag the dimension which is the actually measurement, or statistical count attached to an observation. The `KeyDimension` tages all dimensions which are together uniqly identifying an observation. You can think of them as the *Key* in a realational database.
```

Comment:
> In general we have two mandatory types

mandatory as a dimension must have at least one of them? That's not correct. For example dimension that contains comments would not be a key and also not a measure.

Reply:
Mandatory in the sense of the need to have at least one of both per cube. Can you please rephrase that it makes sense to you?